src/view/gui/fMainWindow.cpp

```diff
@@ -8756,7 +8756,9 @@ void fMainWindow::ChangeMaskOpacity(int newMaskOpacity) // multiLabel uncomment
       this->mSlicerManagers[i]->GetSlicer(j)->mMask->Modified();
     }
   }
-  this->mSlicerManagers[0]->Render();
+  if (!this->mSlicerManagers.empty()) { // prevent crash if no images are loaded
```

Comment:
@AlexanderGetka-cbica Have you looked at fMainWindow::UpdateRenderWindows() ?
This function may do what you need.